docker-compose.f8ui.yml

```diff
@@ -5,6 +5,8 @@ services:
     container_name: fabric8-planner-platform
     image: fabric8-planner-platform
     # command: npm run test:unit
+    environment:
+      PROXY_PASS_URL: "https://api.free-int.openshift.com:443"
```

Comment:
Can you please add a comment regarding why we need this env variable? Maybe add the link to issue/comment which explains why we need this.

Reply:
How about now?

Reply:
Can you please add it above the `PROXY_PASS_URL: "https://api.free-int.openshift.com:443` line ?

Reply:
Done!